Directory.Build.props

```diff
@@ -29,11 +29,11 @@ See https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?v
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.CodeAnalysis.Compilers" Version="4.4.0" >
+    <PackageReference Include="Microsoft.CodeAnalysis.Compilers" Version="4.6.0" >
```

Comment:
@pauld do you know why we were forced to update the version?

Reply:
Hey Wali, unfortunately I did not have time to try and investigate much around the reason that we had to update these. I suspect that it was due to a dotnet version update, but I can't be sure right now